### PR TITLE
055_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_navi_app_NONE_RPC_…

### PR DESCRIPTION
**Pull request by istoimenova**
Script is part of TS: #932

### CODE REVIEW

**Comment by istoimenova** [comment]
@OVikhrov, @dboltovskyi, please review.

File: test_scripts/API/SetAudioStreamingIndicator/055_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_navi_app_NONE_RPC_HMI_level_not_included.lua [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+testCasesForPolicyTable:precondition_updatePolicy_AllowFunctionInHmiLeves({"LIMITED", "BACKGROUND", "FULL"}, "SetAudioStreamingIndicator")
+commonSteps:DeleteLogsFiles()
+
+--[[ General Settings for configuration ]]
+Test = require('connecttest')
+require('user_modules/AppTypes')
+
+--[[ Test ]]
+commonFunctions:newTestCasesGroup("Test")
+
```
Comment (OVikhrov) [inline test_scripts/API/SetAudioStreamingIndicator/055_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_navi_app_NONE_RPC_HMI_level_not_included.lua]: @istoimenova Please activate app and consent device and after that switch app to NONE, In Other case DISALLOWED reason will be unconsented device but not "not allowed HMI level"

Comment (istoimenova) [inline test_scripts/API/SetAudioStreamingIndicator/055_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_navi_app_NONE_RPC_HMI_level_not_included.lua]: @OVikhrov, "unconsented device" is applicable only for External Proprietary flow. The script is prepared for default SDL build: Proprietary.
Rejected.

Comment (OVikhrov) [inline test_scripts/API/SetAudioStreamingIndicator/055_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_navi_app_NONE_RPC_HMI_level_not_included.lua]: @istoimenova agree

**Pull request merged**